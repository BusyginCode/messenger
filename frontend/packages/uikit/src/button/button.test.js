import React from 'react'
import { render } from 'enzyme'
import Button from '../button'

describe('uikit/Button', () => {
  it('should renders correctly', () => {
    const button = render(<Button iconLeft={<span />} />)
    expect(button).toMatchSnapshot()
  })

  it('should renders correctly with component', () => {
    const button = render(<Button component="a" iconRight={<span />} />)
    expect(button).toMatchSnapshot()
  })
})
