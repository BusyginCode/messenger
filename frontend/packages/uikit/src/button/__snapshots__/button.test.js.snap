// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/Button should renders correctly 1`] = `
<button
  class="button primary m navigational"
  type="button"
>
  <span
    class="box iconLeft"
  >
    <span />
  </span>
</button>
`;

exports[`uikit/Button should renders correctly with component 1`] = `
<a
  class="button primary m navigational"
>
  <span
    class="box iconRight"
  >
    <span />
  </span>
</a>
`;
